#### 1)What is the result of the following expression 

In [1]:
5+3*2

11

#### The result is 11

#### 2)What is the output of the following code?

In [2]:
x=5
y=10
x+=y
print(x)

15


#### The output is 15.

#### 3)Evaluate the following boolean expression

In [3]:
(10>5) and (not(5>10))

True

#### Its True

#### 4)What is the result of following division? 

In [4]:
7/2

3.5

#### The result is 3.5

#### 5)What will be value and type of 'x' after the following operation?

In [8]:
x=4//2
type(x)

int

In [9]:
print(x)

2


#### Value of x is 2, Type of x is Integer.

#### 6)What is the output of following code?

In [10]:
a=[1,2,3]
b=a
b[0]=4
print(a)

[4, 2, 3]


#### The output of this code is [4,2,3]

#### 7)What does the following slice operation return?

In [11]:
s='hello'
print(s[1:4])

ell


#### The slice operation returns "ell"

#### 8)What will be the value of 'result'?

In [12]:
result=2 ** 3 ** 2
print(result)

512


#### The value of result is 512

#### 9)Given the list a=[1,2,3,4,5],what will be the result of a[::-1]?

In [14]:
a=[1,2,3,4,5]
print(a[::-1])

[5, 4, 3, 2, 1]


#### The result of a[::-1] is [5, 4, 3, 2, 1]

#### 10)Evaluate the following expression

In [15]:
10%3

1

#### Answer of 10th question is 1.

#### 11)What is the result of following code?

In [16]:
x=[1,2,3]
y=[1,2,3]
print(x==y)
print(x is y)

True
False


#### print(x==y)is True, print(x is y) is False 

#### 12)What  is output of following code?

In [18]:
x='Python'
y=x*3
print(y)

PythonPythonPython


#### The output of the following code is PythonPythonPython

#### 13)What is type and value of x?

In [21]:
x=True+1
print(x)

2


In [22]:
type(x)

int

#### The type is integer and value is 2.

#### 14)output of the following comparison?

In [23]:
print('abc'<'abd')

True


#### The output is True!

#### 15)What is result of following code?

In [24]:
d={'a':1,'b':2}
d['c']=3
print(d)

{'a': 1, 'b': 2, 'c': 3}


#### The result of code is {'a': 1, 'b': 2, 'c': 3}

#### 16)Value of a?

In [26]:
a='123'
a+='456'
print(a)

123456


#### The value of a is 123456

#### 17)What is output of following code?

In [27]:
a=[1,2,3]
b=a[:]
b.append(4)
print(a)
print(b)

[1, 2, 3]
[1, 2, 3, 4]


#### a is [1,2,3] b is [1,2,3,4]

#### 18)Eavluate!

In [29]:
3 * (2 + 4) ** 2

108

#### Answer is 108.

#### 19)Result of following expression?

In [30]:
print(5 == 5.0)

True


#### It shows True.

#### 20)output of following code!?

In [31]:
x=(1,2,3)
y=x+(4,)
print(y)

(1, 2, 3, 4)


#### Output is (1, 2, 3, 4)